data/language/en-GB.txt

```diff
@@ -4114,8 +4114,9 @@ STR_5805    :{SMALLFONT}{BLACK}If checked, your server will be added to the{NEWL
 STR_5807    :{WINDOW_COLOUR_2}Number of rides: {BLACK}{COMMA16}
 STR_5808    :{WINDOW_COLOUR_2}Number of shops and stalls: {BLACK}{COMMA16}
 STR_5809    :{WINDOW_COLOUR_2}Number of information kiosks and other facilities: {BLACK}{COMMA16}
-
 STR_5810    :Disable train length limit
+STR_5811    :{OPENQUOTES}{STRING}{ENDQUOTES}
```

Comment:
The empty line on 4117 has been removed and the STR_5811 are now in use from PR #3480